code/modules/clothing/spacesuits/flightsuit.dm

```diff
@@ -447,11 +447,10 @@
 	else if(crashdir == EAST || crashdir == WEST)
 		V = "x"
 	var/crashpower = 0
-	switch(V)
-		if("y")
-			crashpower = momentum_speed_y
-		if("x")
-			crashpower = momentum_speed_x
+	if(V == "y")
```

Comment:
I mean the whole intermidate V variable is redundant, it's not used anywhere but here so you might as well use the if(crashdir) blocks.